src/rcore.c

```diff
@@ -3829,8 +3829,26 @@ float GetMouseWheelMove(void)
 #if defined(PLATFORM_ANDROID)
     return 0.0f;
 #endif
+    Vector2 wm = CORE.Input.Mouse.currentWheelMove;
+    float result;
+    if (fabs(wm.x) > fabs(wm.y)) result = (float)wm.x;
+    else result = (float)wm.y;
+
 #if defined(PLATFORM_WEB)
-    return CORE.Input.Mouse.currentWheelMove/100.0f;
+    return result/100.0f;
+#endif
+
+    return result;
+}
+
+// Get mouse wheel movement X/Y as a vector
+Vector2 GetMouseWheelMoveV(void)
+{
+#if defined(PLATFORM_ANDROID)
+    return Vector2Zero();
```

Comment:
Please, I prefer to minimize `raymath.h` usage. For those simple functions a return `(Vector2){ 0.0f, 0.0f }` or a manual scaling of values should work.